core-primitives/node-api/api-client-types/src/lib.rs

```diff
@@ -59,21 +78,21 @@ pub use api::*;
 
 #[cfg(feature = "std")]
 mod api {
-	use super::{PairSignature, ParentchainExtrinsicParams, StaticExtrinsicSigner};
+	use super::ParentchainRuntimeConfig;
 	use sp_runtime::generic::SignedBlock as GenericSignedBlock;
 	use substrate_api_client::Api;
 
+	// We should probably switch to the opaque block, then we can get rid of the
+	// runtime dependency here.
+	// pub use itp_types::Block;
 	pub use my_node_runtime::{Block, Runtime, UncheckedExtrinsic};
 
 	pub use substrate_api_client::{
 		api::Error as ApiClientError,
-		rpc::{Error as RpcClientError, WsRpcClient},
+		rpc::{tungstenite_client::TungsteniteRpcClient, Error as RpcClientError},
 	};
 
 	pub type SignedBlock = GenericSignedBlock<Block>;
-	pub type ParentchainExtrinsicSigner =
-		StaticExtrinsicSigner<sp_core::sr25519::Pair, PairSignature>;
```

Comment:
Type is no longer needed, the api-client can simply be instantiated with `key_pair.into()`